utils/build/docker/nodejs/parametric/server.js

```diff
@@ -285,6 +285,11 @@ app.post('/trace/otel/set_attributes', (req, res) => {
   res.json({});
 });
 
+app.get('/trace/config', (req, res) => {
+  const dummyTracer = require('dd-trace').init()
+  res.json( { config: dummyTracer._tracer._config, language: 'nodejs' });
```

Comment:
Instead of returning the whole config object can we return just a set of key's and values. This will allow us to standardize the return value across languages. 

Ex:
```suggestion
  config = dummyTracer._tracer._config
  res.json( { "tracer_enabled": config._tracer_enabled,  "propagation_styles": config._propagation_styles, ...... });
```

We can add to this list as needed.

Reply:
+1, I think it would be cleaner to move the translation logic into the test app so the test runner/test cases don't have that complexity